rector.php

```diff
@@ -2,26 +2,21 @@
 
 declare(strict_types=1);
 
-use Rector\CodeQuality\Rector\ClassMethod\DateTimeToDateTimeInterfaceRector;
 use Rector\Config\RectorConfig;
-use Rector\Core\Configuration\Option;
 use Rector\Core\ValueObject\PhpVersion;
 use Rector\Set\ValueObject\LevelSetList;
 use Rector\Set\ValueObject\SetList;
 
 return static function (RectorConfig $rectorConfig): void {
     $rectorConfig->parallel();
     $rectorConfig->paths([__DIR__.'/src', __DIR__.'/tests']);
-    $rectorConfig->phpVersion(PhpVersion::PHP_74);
+    $rectorConfig->phpVersion(PhpVersion::PHP_80);
     $rectorConfig->phpstanConfig(__DIR__.'/phpstan-rector.neon');
-    $rectorConfig->skip([
-        DateTimeToDateTimeInterfaceRector::class,
-    ]);
```

Comment:
Class no longer exists

Reply:
Might want to add that file to phpstan analysis :see_no_evil:

Reply:
Yeah good idea, done in https://github.com/Lendable/json-serializer/pull/270/commits/7c5c4f3379b78828015ff0c58632e5bcce9bf0fc